plugins/bazaar/src/components/SortView/SortView.tsx

```diff
@@ -116,10 +119,35 @@ export const SortView = (props: SortViewProps) => {
 
   const getSearchResults = () => {
     return bazaarProjects.value
-      ?.filter(project => project.title.includes(searchValue))
+      ?.filter(project =>
+        project.title
+          .toLocaleLowerCase('en-US')
+          .includes(searchValue.toLocaleLowerCase('en-US')),
+      )
       .sort(sortMethods[sortMethodNbr]);
   };
 
+  const { allowed: allowAddProject } = usePermission({
+    permission: bazaarAddPermission,
+  });
+
+  const getSupportItems = () => {
+    const bazaarConfig = config?.getOptionalConfig('bazaar');
+    if (bazaarConfig) {
+      return bazaarConfig.getConfigArray('support').flatMap(itemConf => ({
```

Comment:
this will break as `bazaar.support` is optional, right?

```suggestion
      return bazaarConfig.getOptionalConfigArray('support').flatMap(itemConf => ({
```